Update web-analytics.md (#438)
amplitude/amplitude-docs: content/collections/analytics/en/web-analytics.md

   * For *Breakdown*, choose the default channels you want to display, as well as your campaign and page engagement properties.
   * Set *Conversion Funnel* settings by choosing the events that make up your funnel.

> If *Goals*, choose the key events or metrics you'd like to set up as additional metrics. Goals can be broken down by channels, campaigns, or pages.

Reviewer: ```suggestion
If *Goals*, choose the key events or metrics you'd like to set up as additional metrics. Break down goals by  channels, campaigns, or pages.
```
